analyzer/codechecker_analyzer/analyzers/clangsa/ctu_triple_arch.py

```diff
@@ -22,11 +22,22 @@ def get_compile_command(action, config, source='', output=''):
     for other operations. """
 
     cmd = [config.analyzer_binary]
-    if action.target != "":
-        cmd.append("--target=" + action.target)
+
+    def has_flag(flag):
```

Comment:
It is a small function, but we could have a unittest for it, or a comment about an example param and return value.